hikari/impl/rest.py

```diff
@@ -680,7 +680,7 @@ def __exit__(
             return None
 
     @typing.final
-    async def _request(
+    async def _request(  # noqa: C901 CFQ001
```

Comment:
```suggestion
    async def _request(  # noqa: C901 CFQ001
```
If these are ignored, they should also be documented above